pkg/storage/bench_test.go

```diff
@@ -57,20 +57,29 @@ func BenchmarkMVCCGarbageCollect(b *testing.B) {
 	keySizes := []int{128}
 	valSizes := []int{128}
 	numKeys := []int{1, 1024}
+
 	versionConfigs := []struct {
-		total    int
-		toDelete []int
+		total              int
+		rangeTombstoneKeys []int
+		toDelete           []int
 	}{
-		{2, []int{1}},
-		{1024, []int{1, 16, 32, 512, 1015, 1023}},
+		{
+			total:              2,
+			rangeTombstoneKeys: []int{0, 1, 2},
+			toDelete:           []int{1},
+		},
+		{
+			total:              1024,
+			rangeTombstoneKeys: []int{0, 1, 16, 32},
```

Comment:
Maybe do 0,1,100 instead. We use that a standard number of range keys in all other benchmarks, so it might be nice to standardize on that to better compare the slowdown across multiple benchmarks -- for example, we can know what the best-case slowdown would be from a very simple iterator benchmark, or compare against `MVCCScan`.